x-pack/plugins/actions/server/lib/task_runner_factory.ts

```diff
@@ -154,10 +155,10 @@ export class TaskRunnerFactory {
           );
           // Task manager error handler only kicks in when an error thrown (at this time)
           // So what we have to do is throw when the return status is `error`.
```

Comment:
can we update this comment?

Reply:
Resolved in this commit: https://github.com/elastic/kibana/commit/3f5fbd14c06b90082cad8c953e018a16213cf615